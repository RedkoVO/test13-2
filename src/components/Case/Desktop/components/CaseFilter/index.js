import React from 'react'
import PropTypes from 'prop-types'
import cn from 'classnames'
import withStyles from '@material-ui/core/styles/withStyles'

import styles from './styles'

const CaseFilter = ({ classes }) => (
  <div className={classes.root}>
    <input type="text" className={classes.caseFilterSearch} placeholder="Search" />
    <div className={classes.caseFilterOptions}>
      <div className={classes.caseFilterType}>
        <div className={classes.caseFilterItem}>less than $15&lt;</div>
        <div className={classes.caseFilterItem}>$15-30</div>
        <div className={cn(classes.caseFilterItem, classes.selectedItem)}> $30-60</div>
        <div className={classes.caseFilterItem}>$60-90</div>
        <div className={classes.caseFilterItem}>less than $15&lt;</div>
      </div>
      <div className={classes.caseFilterType}>
        <div className={classes.caseFilterItem}>CS:GO</div>
        <div className={classes.caseFilterItem}>Dota 2</div>
        <div className={cn(classes.caseFilterItem, classes.selectedItem)}>Starcraft 2</div>
        <div className={classes.caseFilterItem}> Maincraft</div>
        <div className={classes.caseFilterItem}>Rainbow Six</div>
      </div>
    </div>
    <div className={classes.caseFilterBtn} />
  </div>
)

CaseFilter.propTypes = {
  classes: PropTypes.object
}

export default withStyles(styles)(CaseFilter)
