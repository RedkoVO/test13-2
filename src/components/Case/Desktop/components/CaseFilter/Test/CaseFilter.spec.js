import * as React from 'react'
import renderer from 'react-test-renderer'

import CaseFilter from '../'

it('renders correctly', () => {
  const tree = renderer.create(<CaseFilter />).toJSON()
  expect(tree).toMatchSnapshot()
})
