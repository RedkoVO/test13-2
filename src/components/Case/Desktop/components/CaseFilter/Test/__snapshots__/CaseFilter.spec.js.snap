// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="CaseFilter-root-1"
>
  <input
    className="CaseFilter-caseFilterSearch-2"
    placeholder="Search"
    type="text"
  />
  <div
    className="CaseFilter-caseFilterOptions-3"
  >
    <div
      className="CaseFilter-caseFilterType-4"
    >
      <div
        className="CaseFilter-caseFilterItem-5"
      >
        less than $15&lt;
      </div>
      <div
        className="CaseFilter-caseFilterItem-5"
      >
        $15-30
      </div>
      <div
        className="CaseFilter-caseFilterItem-5 CaseFilter-selectedItem-7"
      >
         $30-60
      </div>
      <div
        className="CaseFilter-caseFilterItem-5"
      >
        $60-90
      </div>
      <div
        className="CaseFilter-caseFilterItem-5"
      >
        less than $15&lt;
      </div>
    </div>
    <div
      className="CaseFilter-caseFilterType-4"
    >
      <div
        className="CaseFilter-caseFilterItem-5"
      >
        CS:GO
      </div>
      <div
        className="CaseFilter-caseFilterItem-5"
      >
        Dota 2
      </div>
      <div
        className="CaseFilter-caseFilterItem-5 CaseFilter-selectedItem-7"
      >
        Starcraft 2
      </div>
      <div
        className="CaseFilter-caseFilterItem-5"
      >
         Maincraft
      </div>
      <div
        className="CaseFilter-caseFilterItem-5"
      >
        Rainbow Six
      </div>
    </div>
  </div>
  <div
    className="CaseFilter-caseFilterBtn-8"
  />
</div>
`;
